src/postman/util.ts

```diff
@@ -24,5 +24,4 @@ export function traverseItemsAndGroups(
 ) {
   itemGroup.forEachItem(itemCallback);
   if (itemGroupCallback) itemGroup.forEachItemGroup(itemGroupCallback);
-  itemGroup.forEachItemGroup(ig => traverseItemsAndGroups(ig, itemCallback, itemGroupCallback));
```

Comment:
Why is the traversing code changing? Does not really add up

Reply:
Explained here https://github.com/stoplightio/http-spec/pull/68/files#r381923071

Reply:
Aha I understand now! And I am quite sure I told you this already in a previous PR :trollface: 

Good!

Reply:
:P